test/1.4/errors/T_EREGOL.dml

```diff
@@ -69,3 +69,50 @@ bank b8[i < 2] {
     /// ERROR EIDXVAR
     register r size 4 @ i;
 }
+
+// Banks under groups and subdevices should also be subject to overlap checks
+group g[i < 2] {
+    bank b1 {
+        /// ERROR EREGOL
+        register a size 4 @ 8;
+        /// ERROR EREGOL
+        register b size 4 @ 11;
+        // no error
+        register c size 4 @ 18;
+        register d size 4 @ 22;
+    }
+
+    bank b2 {
+        /// ERROR EIDXVAR
+        register r size 4 @ i;
+    }
+}
+
+subdevice sd[i < 2] {
+    bank b {
+        /// ERROR EREGOL
+        register a size 4 @ 8;
+        /// ERROR EREGOL
+        register b size 4 @ 11;
+        // no error
+        register c size 4 @ 18;
+        register d size 4 @ 22;
+    }
+
+    group g {
+        bank b1 {
+            /// ERROR EREGOL
+            register a size 4 @ 8;
+            /// ERROR EREGOL
+            register b size 4 @ 11;
+            // no error
+            register c size 4 @ 18;
+            register d size 4 @ 22;
+        }
+
+        bank b2 {
+            /// ERROR EIDXVAR
+            register r size 4 @ i;
+        }
```

Comment:
many lines for little coverage. The "no error" cases are known from before, and if `group { subdev { group { bank` works then it should be safe to assume the others work too.

Also, I think you can merge b1 and b2 now, after your try/except thingy.

Reply:
Yeah alright. I'll use my discretion though, probably won't reduce it as much as you figure it deserves.